src/material-examples/autocomplete-display/autocomplete-display-example.css

```diff
@@ -0,0 +1,7 @@
+.example-form {
+  width: 500px;
+}
```

Comment:
@amcdnl Think you could add `max-width: 100%` before it merges, here and elsewhere? If not, I'll be happy to in subsequent pr.

https://github.com/angular/material2/pull/5839